src/freenet/pluginmanager/PluginManager.java

```diff
@@ -898,39 +901,46 @@ public PluginInfoWrapper getPluginInfo(String plugname) {
 		return null;
 	}
 
-	/**
-	 * @param pluginClassName The name of the main class of the plugin - that is the class which implements {@link FredPlugin}.
-	 * @return The plugin with the given class, or null if no matching plugin was found.
-	 */
-	public PluginInfoWrapper getPluginInfoByClassName(String pluginClassName) {
-		synchronized(pluginWrappers) {
-			for(PluginInfoWrapper piw : pluginWrappers) {
-				if(piw.getPluginClassName().equals(pluginClassName))
-					return piw;
-			}
-		}
-		return null;
-	}
-
-	/**
-	 * @param pluginFileName The filename of the JAR from which the plugin was loaded.
-	 * @return Null if no matching plugin was found.
-	 */
-	public PluginInfoWrapper getPluginInfoByFileName(String pluginFileName) {
-		synchronized(pluginWrappers) {
-			for(PluginInfoWrapper piw : pluginWrappers) {
-				if(piw.getFilename().equals(pluginFileName))
-					return piw;
-			}
-		}
-		return null;
-	}
+    /**
+     * @param pluginClassName
+     *            The name of the main class of the plugin - that is the class which implements
+     *            {@link FredPlugin}.
+     * @return The plugin with the given class, or null if no matching plugin was found.
+     */
+    public PluginInfoWrapper getPluginInfoByClassName(String pluginClassName) {
+        synchronized(pluginWrappers) {
+            for(PluginInfoWrapper piw : pluginWrappers) {
+                if(piw.getPluginClassName().equals(pluginClassName)) {
+                    return piw;
+                }
+            }
+        }
+        return null;
+    }
+
+    /**
+     * @param pluginFileName
+     *            The filename of the JAR from which the plugin was loaded.
+     * @return Null if no matching plugin was found.
+     */
+    public PluginInfoWrapper getPluginInfoByFileName(String pluginFileName) {
+        synchronized(pluginWrappers) {
+            for(PluginInfoWrapper piw : pluginWrappers) {
+                if(piw.getFilename().equals(pluginFileName)) {
+                    return piw;
+                }
+            }
+        }
+        return null;
+    }
 
 	/**
 	 * look for a FCPPlugin with given classname
 	 * @param plugname
 	 * @return the plugin or null if not found
+     * @deprecated Use {@link #getPluginFCPServer(String)} instead.
```

Comment:
The commits up to commit 84aae9936ac4f1001e74d131d8b37526dbd8dbba add JavaDoc which explains this.